### PR TITLE
added plugin 3D Geeks (id: geeks3d)

### PR DESCRIPTION
**Pull request by 2BWorks**

### CODE REVIEW

**Comment by jneilliii** [comment]
Problems...

1. https://github.com/2BWorks/OctoPrint-3DGeeks/blob/42aa6cf514fda9a3cd36e3d724bfad7670ebfa55/octoprint_geeks3d/__init__.py#L39
  a. Why referencing printoid instead of your own plugin name? Obviously this was just a fork to work with your app, but you should resolve that prior to being accepted.
2. https://github.com/2BWorks/OctoPrint-3DGeeks/blob/42aa6cf514fda9a3cd36e3d724bfad7670ebfa55/octoprint_geeks3d/__init__.py#L129
  a. returned dictionary needs to match your plugin_identifier, including case `geeks3d` instead of  `Geeks3D`.
  b. you might want to change the displayName to match your plugin's name defined later down, as well as in setup.py [here](https://github.com/2BWorks/OctoPrint-3DGeeks/blob/42aa6cf514fda9a3cd36e3d724bfad7670ebfa55/setup.py#L14) for consistency.

**Comment by 2BWorks** [comment]
Thanks! Will resolve asap!

**Comment by 2BWorks** [comment]
@jneilliii Made changes accordingly

File: _plugins/geeks3d.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+Install via the bundled [Plugin Manager](https://docs.octoprint.org/en/master/bundledplugins/pluginmanager.html)
+or manually using this URL:
+
+    https://github.com/2BWorks/OctoPrint-3DGeeks/archive/master.zip
+
+
+## Usage
+### Instance configuration
+After installation restart OctoPrint, and go to the Settings tab. You should see a new tab named 3D Geeks, go ahead and click that tab. You should be presented with a QR Code.
```
Comment (jneilliii) [inline _plugins/geeks3d.md:83]: Would you mind changing this? Settings is not really a tab and might be confusing for end-users.

`After installation restart OctoPrint, and go to Settings. You should see a new section in plugins named 3D Geeks, go to that section and you should be presented with a QR Code.

File: _plugins/geeks3d.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+On your Android powered device open the 3D Geeks app, go to:
+```
+Settings > OctoPrint settings
+```
+Press the `+`-Icon in the bottom-right hand corner.
+
+Press the scan icon, scan the QR code you're being presented. If everything goes well all the nesecary fields will be filled in automatically for you. You can now press Test connection for a quick test of the connection to your OctoPrint instance.
```
Comment (jneilliii) [inline _plugins/geeks3d.md:91]: typo: should be `necessary`

File: _plugins/geeks3d.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+NOTE: Be sure you're Android device is connected to the same network as your OctoPrint instance, as they communicate through the local network. When the test succeeds you can press Create. Your OctoPrint instance is now saved and can receive files from 3D Geeks.
+
+NOTE: You can create as much OctoPrint instances as you desire. When uploading a file, the app will ask you which instance to upload to.
```
Comment (jneilliii) [inline _plugins/geeks3d.md:95]: grammar: should be `many` not `much`.

Comment (2BWorks) [reply _plugins/geeks3d.md]: done

File: _plugins/geeks3d.md (hunk 2 above)
Comment (2BWorks) [reply _plugins/geeks3d.md]: done

File: _plugins/geeks3d.md (hunk 1 above)
Comment (2BWorks) [reply _plugins/geeks3d.md:83]: done

**Comment by 2BWorks** [comment]
@jneilliii I think the check has failed unrightfully. How can we spin it off again?

**Comment by foosel** [comment]
Manually fixed the python compatibility flag - it was not indented probably and thus not taken up correctly.

**Pull request merged**